test/step-definitions/requests.ts

```diff
@@ -27,13 +29,20 @@ export class requests {
 
   @given(/make a GET request to "([^"]*)"/)
   public async getRequest(url: string) {
+    if (/\/{id}/.test(url)) {
+      url = url.replace(/{id}/, this.context.documentId);
+    }
```

Comment:
I think this is the same for multiple requests, could it be moved to a private method?

Reply:
Sure 👍🏽